recipes-bsp/imx-mkimage/imx-mkimage_git.inc

```diff
@@ -4,6 +4,7 @@ DEPENDS = "zlib-native openssl-native"
 
 SRC_URI = "git://github.com/nxp-imx/imx-mkimage.git;protocol=https;branch=${SRCBRANCH} \
            file://0001-iMX8M-soc.mak-use-native-mkimage-from-sysroot.patch \
+           file://0001-iMX9-soc.mak-dynamically-set-AHAB_IMG.patch \
```

Comment:
Hey @thochstein, could you please ensure that you apply those changes for the upcoming release? Thank you!

Reply:
Hello,

I'm going to test this tomorrow morning, but I was getting the very same error (which I'll try to reproduce and send the logs here) even though I was setting IMX_SOC_REV to "A1" by hand in my local.conf, so I don't really see how setting this elsewhere would give a different result. 

Kind regards,
João

Reply:
Indeed, I did run into the error as well. It seems that imx-mkimage is not up to date either.

Reply:
Hopefully this is fixed now by #1701.

Reply:
Hi @jmarcoscosta,

Glad to hear it works!

The firmware name is not hard-coded anymore [1] and is driven by `IMX_SOC_REV`, which sets `SECO_FIRMWARE_NAME` and gets passed to `imx-mkimage` [2].

[1] https://github.com/Freescale/meta-freescale/commit/0688f79193e697b1bd7444358a7b46c22b83ca2b
[2] https://github.com/nxp-imx/imx-mkimage/blob/lf-6.1.36_2.1.0/iMX9/soc.mak#L20-L21

In the end this error should never have happened had we done the upstreaming of NXP release 6.1.36-2.1.0 properly. I can try to flesh out the new commit messages with some of this information.

Reply:
I see, thanks!

Could you please give me a heads-up once this is all merged into meta-freescale's master?